jenkins/pipelines/cd/atom-jobs/build-common.groovy

```diff
@@ -838,33 +838,62 @@ cp audit/audit-1.so.md5 ${TARGET}/bin
 cp audit/audit-1.so ${TARGET}/bin
 """
 
+def usePod(){
+    return OS=="linux"
+}
+
+def upload_fileserver(local, remote){
+    if(usePod()){
+        container("ks3util"){
+            withCredentials([file(credentialsId: 'ks3util-secret-config', variable: 'KS3UTIL_CONF')]) {
+                sh "ks3util -c \$KS3UTIL_CONF cp -f $local ks3://ee-fileserver/download/${remote}"
+            }
+        }
+    }else{
+        withCredentials([file(credentialsId: 'ks3util-secret-config', variable: 'KS3UTIL_CONF')]) {
+            sh "ks3util -c \$KS3UTIL_CONF cp -f $local ks3://ee-fileserver/download/${remote}"
+        }
+    }
+}
+
+def download_fileserver(remote, local){
+    if(usePod()){
+        container("ks3util"){
+            withCredentials([file(credentialsId: 'ks3util-secret-config', variable: 'KS3UTIL_CONF')]) {
+                sh "ks3util -c \$KS3UTIL_CONF cp -f ks3://ee-fileserver/download/${remote} $local"
```

Comment:
please do not hardcode it, put it to a final variable.

Reply:
final will fail